master/buildbot/test/unit/test_steps_master.py

```diff
@@ -280,3 +280,44 @@ def manipulate(props):
             # the renderer returns renderable!
             return {'a': Interpolate('b')}
         return self.doOneTest(properties=manipulate)
+
+
+class TestAssertRenderable(steps.BuildStepMixin, unittest.TestCase):
+
+    def setUp(self):
+        return self.setUpBuildStep()
+
+    def tearDown(self):
+        return self.tearDownBuildStep()
+
+    def test_lambda_pass(self):
+        self.setupStep(master.AssertRenderable(
+            lambda prop: prop.getProperty("test_prop") == "foo"))
```

Comment:
well, this is not really a renderable, this is a function.
I think it is better to stick with renderable
```python
    self.setupStep(master.Assert(util.renderable(lambda prop:prop.getProperty("test_prop") == "foo"))))
```